packages/graphqlsp/src/autoComplete.ts

```diff
@@ -46,22 +48,29 @@ export function getGraphQLCompletions(
     ? bubbleUpCallExpression(node)
     : bubbleUpTemplate(node);
 
-  let text, cursor;
+  let text, cursor, schemaToUse: GraphQLSchema;
   if (
     ts.isCallExpression(node) &&
     isCallExpression &&
     templates.has(node.expression.getText()) &&
     node.arguments.length > 0 &&
     ts.isNoSubstitutionTemplateLiteral(node.arguments[0])
   ) {
+    const typeChecker = info.languageService.getProgram()?.getTypeChecker();
+    const schemaName = getSchemaName(node, typeChecker);
+
     const foundToken = getToken(node.arguments[0], cursorPosition);
-    if (!schema.current || !foundToken) return undefined;
+    if ((!schema.current && !schema.multi[schemaName]) || !foundToken)
+      return undefined;
 
     const queryText = node.arguments[0].getText().slice(1, -1);
     const fragments = getAllFragments(filename, node, info);
 
     text = `${queryText}\n${fragments.map(x => print(x)).join('\n')}`;
     cursor = new Cursor(foundToken.line, foundToken.start - 1);
+    schemaToUse = schema.multi[schemaName]
+      ? schema.multi[schemaName]!.schema
+      : schema.current!.schema;
```

Comment:
How about merging these two? Seems a bit confusing and error-prone to assert non-nullability at the end here `!`, when we're checking this above.

Above we're also checking `!schema.current && !schema.multi[schemaName]`, which doesn't seem right over checking either of them dependending on `schemaName`.

The fallback here can also become `schemaName ? schema.multi[schemaName] : schema.current` if the return value of `getSchemaName` is fixed to be nullable